dist2src/worker/updater.py

```diff
@@ -34,13 +33,13 @@ def check_updates(
         self, project: Optional[str] = None, branch: Optional[str] = None
     ):
         """
-        Check if repositories in a soource-git namespace need to be updated
+        Check if repositories in a source-git namespace need to be updated
         and create Celery tasks to update them, if configured and needed.
 
         Limit the checks to 'project' and 'branch', if the arguments are
         specified.
         """
-        logger.debug(f"Source-git API: {self.cfg.src_git_svc.api_url!r}")
+        logger.debug(f"Source-git API: {self.cfg.src_git_svc.instance_url!r}")
```

Comment:
The text and the name of the logged value is not in sync. Is this intentional?

Reply:
I'm actually wondering why GitlabService didn't have the api_url set, I'll sync them.

Reply:
yeah, api_url is pagure specific, changing the string to `Source-git instance` then